_events/sr2019/competition.md

```diff
@@ -12,9 +12,32 @@ The Student Robotics Competition marks the ultimate stage of SR2019, showcasing
 
 The competition will be held from *Saturday 6th April* until *Sunday 7th April*. Doors open 9am and we aim to finish around 6pm.
 
-The Saturday will begin with a briefing on the rules and general struture of the weekend, followed by some time for teams to test their robots in the arena. Soon after, the league matches will start, in which your robot will compete against 3 other bots to rise up the league. These continue for the rest of the day.
-
-On Sunday morning, the league matches will continue. Once these have concluded, we will begin knockout matches, the order of which is determined by your position in the league. The remaining teams will compete in the finals, followed by the awards for the winning teams.
+### Day 1 - 6th April, 2019
+
+
+| Time  | Activity                         |
+|-------|----------------------------------|
+| 9:00  | Doors open                       |
+| 9:30  | Introduction and safety briefing |
+| 9:40  | Tinker time                      |
+| 11:00 | League matches                   |
+| 12:00 | Lunch and Tinker time            |
```

Comment:
We're inconsistent about our capitalisation here and elsewhere. I suggest making all of these "Sentence case"